com.github.phase1geo.minder.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why not just patch the current appdata instead of shipping a new one?

Reply:
Because it's broken. I reported [twice](https://github.com/phase1geo/Minder/pull/96) this but we still can't fix this. Now we got this error when validating:

```
Running appstream-compose
Processing application com.github.phase1geo.minder
Error loading AppData file: failed to parse /app/share/appdata/com.github.phase1geo.minder.appdata.xml: Error on line 161 char 82: <li> already set 'Changed' and tried to replace with ' default filename to use translated value.'
```

Reply:
FYI, I believe that I have all Flatpak related issues fixed in a development branch that will be generally available in the next release, including issues with AppData.

Reply:
Well, this was trivial this time. Fixed. appstream-util don't like `<em>`. I'll wait for @phase1geo before sending PR upstream but now all resolved and ready for Flathub.